edunext_openedx_extensions/microsite_configuration/backends/database.py

```diff
@@ -1,217 +1,12 @@
-
-
-class DatabaseMicrositeTemplateBackend(BaseMicrositeTemplateBackend):
-    """
-    Specialized class to pull templates from the database.
-    This Backend would allow us to save templates in DB and pull
-    them from there when required for a specific microsite.
-    This backend can be enabled by `MICROSITE_TEMPLATE_BACKEND` setting.
-
-    E.g. we have setup a microsite for subdomain `mua.edx.org` and
-    We have a DB entry like this in table created by MicrositeTemplate model.
-
-    microsite = Key for microsite(mua.edx.org)
-    template_uri = about.html
-    template = <html><body>Template from DB</body></html>
-
-    While using DatabaseMicrositeTemplateBackend any request coming from mua.edx.org/about.html
-    would get about.html template from DB and response would be the value of `template` column.
-    """
-    def get_template_path(self, relative_path, **kwargs):
-        return relative_path
-
-    def get_template(self, uri):
-        """
-        Override of the base class for us to look into the
-        database tables for a template definition, if we can't find
-        one we'll return None which means "use default means" (aka filesystem)
-        """
-        cache_key = "template_cache." + fasthash(microsite_get_value('site_domain') + '.' + uri)
-        template_text = cache.get(cache_key)  # pylint: disable=maybe-no-member
-
-        if not template_text:
-            # cache is empty so pull template from DB and fill cache.
-            template_obj = MicrositeTemplate.get_template_for_microsite(
-                microsite_get_value('site_domain'),
-                uri
-            )
-
-            if not template_obj:
-                # We need to set something in the cache to improve performance
-                # of the templates stored in the filesystem as well
-                cache.set(  # pylint: disable=maybe-no-member
-                    cache_key, '##none', settings.MICROSITE_DATABASE_TEMPLATE_CACHE_TTL
-                )
-                return None
-
-            template_text = template_obj.template
-            cache.set(  # pylint: disable=maybe-no-member
-                cache_key, template_text, settings.MICROSITE_DATABASE_TEMPLATE_CACHE_TTL
-            )
-
-        if template_text == '##none':
-            return None
-
-        return Template(
-            text=template_text
-        )
-
-    @staticmethod
-    @receiver(post_save, sender=MicrositeTemplate)
-    def clear_cache(sender, instance, **kwargs):  # pylint: disable=unused-argument
-        """
-        Clear the cached template when the model is saved
-        """
-        cache_key = "template_cache." + fasthash(instance.microsite.subdomain + '.' + instance.template_uri)
-        cache.delete(cache_key)  # pylint: disable=maybe-no-member
-
-
-class EdunextCompatibleDatabaseMicrositeBackend(DatabaseMicrositeBackend):
+class EdunextCompatibleDatabaseMicrositeBackend(BaseMicrositeBackend):
```

Comment:
So the reason why we keep DatabaseMicrositeBackend is because the method get_all_config becuase the rest of the methods of this class are overiden by EdunextCompatibleDatabaseMicrositeBackend. Or maybe to keep the reference